prow/config.yaml

```diff
@@ -4032,6 +4065,28 @@ periodics:
         requests:
           cpu: 4
 
+- name: ci-kubernetes-e2e-kubeadm-dind-ipv6
+  interval: 6h
+  agent: kubernetes
+  labels:
+    preset-service-account: "true"
+  spec:
+    containers:
+    - image: diverdane/e2e-kubeadm-dind:v20180814-829bf6a-dirty
+      args:
+      - "--repo=k8s.io/kubernetes=master"
+      - "--repo=github.com/leblancd/kubeadm-dind-cluster=dockernet-172.20.0.0"
+      - "--upload=gs://kubernetes-jenkins/logs"
+      - "--git-cache=/root/.cache/git"
+      - "--timeout=75"
+      - "--clean"
+      # Privileged mode required to enable IPv6 and Docker-in-Docker mode
```

Comment:
er, are we changing networking flags on the host vs just container local routing? we should avoid that. In that case we may need to do this on a VM instead :( :man_facepalming:

Reply:
The networking flag net.ipv6.conf.all.disable_ipv6 will only get set to 0 in the container, not on the host.

Reply:
:sweat_smile: thanks, SGTM